release/release-debian

```diff
@@ -121,10 +121,8 @@ EOF
         dpkg-source --extract $DSC $distribution
         sed -i "s/$TAG-0/$TAG-0~$distribution/" $distribution/debian/changelog
         sed -i "s/experimental/$distribution/" $distribution/debian/changelog
-        # Hack: Remove PCP build dependencies while pcp is not in testing
-        # (https://tracker.debian.org/pcp)
-        if [ "$distribution" = unstable ]; then
-            sed -i '/libpcp.*-dev/d' $distribution/debian/control
+        if [ -x $distribution/debian/adjust-for-release ]; then
```

Comment:
maybe add a comment here?

such as: `# Call distribution specific release scripts if present, e.g. for hacks`